spec/api/v1/auth_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "Auth API", type: :api do
+  let!(:user) { create(:user_with_team, password: "Password1!") }
+  let(:team) { user.teams.first }
+
+  describe "POST /api/v1/auth" do
+    context "when params are valid" do
+      it "returns api token" do
+        params = {
+          email: user.email,
+          password: "Password1!",
+          subdomain: team.subdomain
```

Comment:
For the single site version we won’t even have subdomains for teams, it’s probably best to just take the users email + password and we can figure out the subdomain thing for the multisite version later?